DQM/CTPPS/python/ctppsDQM_cff.py

```diff
@@ -57,7 +57,7 @@
 
 _ctppsDQMOfflineSource = cms.Sequence(
   ctppsPixelDQMOfflineSource
-  + ctppsDiamondDQMSource
```

Comment:
@vavati this PR is modifying the `_ctppsDQMOfflineSource` to include `CTPPSDiamondDQMSource.cc` which is also modified here so that it requires `LHCInfo` conditions. So in order to test this PR new GTs will be needed. I don't see how you can test it without modifying the GT (even if you run on run2 data)

Reply:
My point is: if you have a single ctpps dqm sequence and you use it for both MC and Data you will have to provide all the proper tags in all GTs. If you don't care about ctpps in the MC sequence then you should probably split (or customize) the sequence in a different way for data and for MC. 

I hope I understood the issue correctly. Maybe @jfernan2 can comment/suggest as well.

Reply:
I totally agree with @francescobrivio 
We cannot approve a PR which is not going to run in production....